portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/ItemSelector.path

```diff
@@ -53,6 +53,12 @@
 	<td>//div[contains(@data-qa-id,'rowItemContent') and contains(.,'${key_title}')]//span[contains(.,'${key_locationName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>LOCATION_INFO_LIST_TITLE</td>
+	<td>//div[contains(@data-qa-id,'rowItemContent')]//a[contains(.,'${key_locationName}')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @heiditrindade, I think that you could create this path inside PRM dependencies.

Reply:
Hi @AfonsoHSantos, I thought about it, but I decided to create that path there because there's another path right above the one I created that can find the role's description, but not the title, so, they are similar. Also, I thought that other people can find it easier if someone ever needs to assert a role title if the path it's not inside PRM dependencies.